load/peering.go

```diff
@@ -0,0 +1,354 @@
+package load
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	petname "github.com/dustinkirkland/golang-petname"
+	"github.com/hashicorp/consul/api"
+	"github.com/mkeeler/consul-load-test/metrics"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/time/rate"
+)
+
+const (
+	// acceptorPeerName is the peer name used by the peering clusters
+	// the reverse will be <peering address>-<port>
+	acceptorPeerName = "acceptor"
+)
+
+// KVConfig is the configuration for the KV load generator
+type PeeringConfig struct {
+	// UpdateRate is the number of KV updates per second
```

Comment:
I think this is being used a little differently than in the KV case. Is this more accurate?
```suggestion
	// UpdateRate time in seconds between peered service registrations
```